yb-voyager/src/utils/utils.go

```diff
@@ -444,3 +444,24 @@ func GetFSUtilizationPercentage(path string) (int, error) {
 	percUtilization := 100 - int((stats.Bavail*100)/stats.Blocks)
 	return percUtilization, nil
 }
+
+//read the file and return slice of csv 
+func ReadFileToSlice(listFilePath string) []string {
+	file, err := os.Open(listFilePath)
+	if err != nil {
+		ErrExit("error opening file %s: %v", listFilePath, err)
```

Comment:
Return error from the function. We should stop introducing more `ErrExit()` calls. Let the caller deal with the error.